unime/src/lib/credentials/CredentialList.svelte

```diff
@@ -98,9 +98,7 @@
     {#each credentials as credential}
       <ListItemCard
         id={credential.id}
-        title={credential.metadata.display.name ??
-          credential.data.credentialSubject.achievement?.name ??
-          credential.data.type.at(-1)}
+        title={credential.display_name}
```

Comment:
Are we still making sure that there is _something_ displayed in any case? If `display_name` can be an empty string, then we won't see anything in the UI, right?

Reply:
Yes because this is being added in the backend. Its part of the previous PR, we talked about few weeks ago where I said I would move this to the backend.

Reply:
Maybe I commented in the wrong place (frontend), but would we handle `display_name` to never be an empty string?

Reply:
Well it can be an empty string but the whole idea of a display_name is that it's always set right? I think we should require that some fields are always set. Because to have a name missing of a credential in my opinion is not a correct state. Because to the end user it's hard to make sense of what the credential is otherwise right

Reply:
Yes, correct. That's what I was asking. We should make sure it can never be empty in the backend. We do that, right?

Reply:
I just moved those three ?? fields to the backend. If something is still missing it would have been null or empty before as well, so maybe for another story. To see where its added now: identity-wallet/src/verifiable_credential_record.rs